.github/workflows/docker.yml

```diff
@@ -21,19 +21,30 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v3
 
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@818d4b7b91585d195f67373fd9cb0332e31a7175
+      with:
+        images: ${{ env.REGISTRY }}/cosmos/${{ env.IMAGE_NAME }}
+
+    - name: Build and push Docker image
```

Comment:
this task isn't pushing anymore right? We can update the name

Reply:
yeah, good point!